.jenkins/common_job_properties.groovy

```diff
@@ -100,14 +100,23 @@ class common_job_properties {
   // below to insulate callers from internal parameter defaults.
   private static def setPullRequestBuildTrigger(def context,
                                                 def commitStatusContext,
-                                                def successComment = '--none--') {
+                                                def successComment = '--none--',
+                                                def trigger_phrase = '') {
     context.triggers {
       githubPullRequest {
         admins(['asfbot'])
         useGitHubHooks()
         orgWhitelist(['apache'])
         allowMembersOfWhitelistedOrgsAsAdmin()
         permitAll()
+        // trigger_phrase is the argument which gets set when we want to allow
+        // post-commit builds to run against pending pull requests. This block
+        // overrides the default trigger phrase with the new one and sets the
+        // build to happen only when it sees the trigger phrase.
```

Comment:
rm " sets the build to happen only when it sees the trigger phrase". Say 'setting this will disable automatic invocation of this build; it will require the phrase to start it"